slurm/install/install.py

```diff
@@ -86,8 +85,8 @@ def setup_users(s: InstallSettings) -> None:
     )
 
 
-def run_installer(path: str, mode: str) -> None:
-    subprocess.check_call([path, mode])
+def run_installer(s: InstallSettings, path: str, mode: str) -> None:
```

Comment:
I overall think we need to be a bit more careful in scripts where we are using relative/absolute paths. right now there are only 2 scripts, but if we plan to add more, then we need to make it look more ansible like. right now shell to python then to back shell is getting a bit convoluted. We can make immediate fixes but something to keep in mind.